pkg/config/loader.go

```diff
@@ -28,5 +30,7 @@ func FromYAML(contents []byte) (*MetricsDiscoveryConfig, error) {
 	if err := yaml.UnmarshalStrict(contents, &cfg); err != nil {
 		return nil, fmt.Errorf("unable to parse metrics discovery config: %v", err)
 	}
+	metrics.Rules.WithLabelValues("normal").Set(float64(len(cfg.Rules)))
```

Comment:
nit: rather call this `"resource"`.

Reply:
happy to adjust. do you mean the specific label should be `resource` instead of `normal`?

Reply:
Yes :+1: as this refers to "resource metrics".